js/src/tab.js

```diff
@@ -168,7 +168,16 @@ class Tab extends BaseComponent {
     event.stopPropagation()// stopPropagation/preventDefault both added to support up/down keys without scrolling the page
     event.preventDefault()
     const isNext = [ARROW_RIGHT_KEY, ARROW_DOWN_KEY].includes(event.key)
-    const nextActiveElement = getNextActiveElement(this._getChildren(), event.target, isNext, true)
+    let nextActiveElement = getNextActiveElement(this._getChildren(), event.target, isNext, true)
```

Comment:
```js

   let nextActiveElement = getNextActiveElement(this._getChildren().filter(!isDisabled), event.target, isNext, true)
```
would work or not? :thinking:

Reply:
Just tested it: `Uncaught TypeError: (!isDisabled) is not a function`. It doesn't work like that but I get the spirit, agree that it would be way better and readable.